plugins/tiddlywiki/webdav/html-div-tiddler.tid

```diff
@@ -0,0 +1,9 @@
+title: $:/core/templates/html-div-tiddler
```

Comment:
IMO this template is essential and only used to store tiddler content in the single-file version. ... It should _not_ be modified, or used in other ways. ... Saving tiddlers in .tid format looks different!

Reply:
Why do you want to change it?